app/models/manageiq/providers/cloud_manager/template.rb

```diff
@@ -17,7 +17,8 @@ def self.eligible_for_provisioning
                             ManageIQ::Providers::Azure::CloudManager::Template
                             ManageIQ::Providers::Google::CloudManager::Template
                             ManageIQ::Providers::Openstack::CloudManager::VolumeTemplate
-                            ManageIQ::Providers::Openstack::CloudManager::VolumeSnapshotTemplate))
+                            ManageIQ::Providers::Openstack::CloudManager::VolumeSnapshotTemplate
+                            ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::Template))
```

Comment:
should this list be alphabetized for clarity?

Reply:
Done.

I was going to ask if we should also sort Templates from Volume* first, but turns out that comes free :)